packages/libdispatch/build.sh

```diff
@@ -1,9 +1,8 @@
 TERMUX_PKG_HOMEPAGE=https://github.com/apple/swift-corelibs-libdspatch
 TERMUX_PKG_DESCRIPTION="The libdispatch project, for concurrency on multicore hardware"
 TERMUX_PKG_LICENSE="Apache-2.0"
-_VERSION=5.1
+_VERSION=5.2
 TERMUX_PKG_VERSION=1:${_VERSION}
```

Comment:
Can this epoch be removed now, or am I stuck with it from here on out?

Reply:
It is permanent, like a version. If removed, package will not be updatable.